### ISSUE TITLE
Implement news details screen

### ISSUE THREAD

**Issue by AkshayChordiya** [AkshayChordiya/News#47]
- Implement a new screen / activity showing details about the clicked news article
- Try to use Jetpack compose for the UI

**Comment by rohithThammaiah** [comment]
Hey, can I take a shot at this?

**Comment by sivansundar** [comment]
Hey! Would love to take a jab at this

**Comment by AkshayChordiya** [comment]
Yeah sure @rohithThammaiah you can give it a shot. @sivansundar feel free to give it a spin too but since @rohithThammaiah was first here, it seems fair to assign him the issue or we can see when PR is created 🙃

**Comment by sivansundar** [comment]
Makes sense. Thanks, @AkshayChordiya :)
